autometrics-cli/src/sloth.rs

```diff
@@ -10,6 +10,21 @@ pub struct Arguments {
     #[clap(long, default_values = &["90", "95", "99", "99.9"])]
     objectives: Vec<String>,
 
+    /// Minimum traffic to trigger alerts in events/second.
+    ///
+    /// If a service (all function sharing the same service name and objective
+    /// percentage) has less than this many calls per second, then it won't
+    /// trigger any alert.
+    ///
+    /// Defaults to "at least 1 event per minute" (which is around
+    /// 0.16666...).
+    ///
+    /// Note that if a SLO has the same service name but with different
+    /// objective targets (as in "objective percentage") then the threshold
+    /// won't behave as expected.
+    #[clap(short, long, default_value_t = 0.16666666)]
```

Comment:
I wonder if it would make more sense to parameter defined as events per minute so the default would just be `1`

Reply:
Why not, I don't have any strong opinion on that one